randovania/games/fusion/logic_database/Sector 3 PYR.txt

```diff
@@ -915,18 +915,26 @@ Extra - room_id: [23, 18]
               Missiles ≥ 15 and 45 DMG Missiles
               Missiles ≥ 25 and 20 DMG Missiles
               Missiles ≥ 45 and Combat (Beginner) and 10 DMG Missiles
+              All of the following:
+                  # Don't miss a missile: https://youtu.be/ejVViu60J_o
+                  Combat (Expert)
+                  Any of the following:
+                      Missiles ≥ 30 and 10 DMG Missiles
+                      Missiles ≥ 15 and 20 DMG Missiles
+                      Missiles ≥ 7 and 45 DMG Missiles
           Any of the following:
               # Health Requirements
-              Normal Damage ≥ 500
+              Normal Damage ≥ 300
               # Fight Boss Damageless: https://youtu.be/ejVViu60J_o
-              Combat (Expert)
-              Combat (Beginner) and Normal Damage ≥ 400
-              Combat (Intermediate) and Normal Damage ≥ 300
-              Combat (Advanced) and Normal Damage ≥ 100
+              Combat (Advanced)
+              Combat (Beginner) and Normal Damage ≥ 240
+              Combat (Intermediate) and Normal Damage ≥ 100
           Any of the following:
               # Dodging Requirements
-              Combat (Advanced) or Normal Damage ≥ 200 or Have Any Jump Upgrade
-              Screw Attack and Combat (Intermediate)
+              Have Any Jump Upgrade
+              # Walljump over BOX / to the monkey bar
+              Combat (Intermediate)
```

Comment:
I feel like this could be kept at advanced to do the fight with just jumping over BOX.
Coupled with the health reqs you'd be expected to dodge and only have 1E which seems rather tight.

Reply:
Dodging is easy tho. You go to the left wall, walljump to the monkey bar and then you cant get hit (if you position yourself right).
If you do somehow get hit, you either walk to the left wall again, or if you cant, walljump over box at the right wall.

Reply:
I think I just suck at combat in this game